src/devices/Display/FontHelper.cs

```diff
@@ -0,0 +1,203 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Iot.Device.Display
+{
+    /// <summary>
+    /// Provides segment mappings for hexadecimal digits and certain ASCII characters
+    /// </summary>
+    public static class FontHelper
+    {
+        #region Private members
+        /// <summary>
+        /// Hexadecimal digit (0..F) fonts
+        /// </summary>
+        private static readonly Font[] s_hexDigits =
+        {
+            Font.Digit_0,
+            Font.Digit_1,
+            Font.Digit_2,
+            Font.Digit_3,
+            Font.Digit_4,
+            Font.Digit_5,
+            Font.Digit_6,
+            Font.Digit_7,
+            Font.Digit_8,
+            Font.Digit_9,
+            Font.Digit_A,
+            Font.Digit_B,
+            Font.Digit_C,
+            Font.Digit_D,
+            Font.Digit_E,
+            Font.Digit_F
+        };
+
+        /// <summary>
+        /// Upper case letter fonts
+        /// </summary>
+        private static readonly Font[] s_upperCaseLetters =
+        {
+            Font.Letter_A,
+            Font.Letter_B,
+            Font.Letter_C,
+            Font.Letter_D,
+            Font.Letter_E,
+            Font.Letter_F,
+            Font.Letter_G,
+            Font.Letter_H,
+            Font.Letter_I,
+            Font.Letter_J,
+            Font.Whitespace, // letter K is not supported
+            Font.Letter_L,
+            Font.Whitespace, // letter M is not supported
+            Font.Letter_N,
+            Font.Letter_O,
+            Font.Letter_P,
+            Font.Whitespace, // letter Q is not supported
+            Font.Letter_R,
+            Font.Letter_S,
+            Font.Whitespace, // letter T is not supported
+            Font.Letter_U,
+            Font.Whitespace, // letter V is not supported
```

Comment:
U is close enough :P (better than whitespace)

Reply:
treat as optional

Reply:
I used wikipedia mostly for inspiration:
https://en.wikipedia.org/wiki/Seven-segment_display

I deliberately left out fonts that would've been ambiguous from this helper, except for certain obvious ones we can't really live without (1, I, l, |, etc.).